src/tasks/AndroidAppBuilder/Templates/monodroid.c

```diff
@@ -304,7 +307,11 @@ mono_droid_runtime_init (const char* executable, int managed_argc, char* managed
     mono_jit_set_aot_mode(MONO_AOT_MODE_NORMAL);
 #endif // FULL_AOT
 #endif // FORCE_INTERPRETER
+}
 
+static int 
+mono_droid_execute_assembly (const char *executable, int managed_argc, char* managed_argv[])
+{    
     MonoDomain *domain = mono_jit_init_version ("dotnet.android", "mobile");
```

Comment:
I think we can move mono_jit_init_version and mono_droid_load_assembly into the runtime init too, those don't execute user code and I'd count them as startup.

Reply:
I thought keeping it with `mono_droid_execute_assembly` because the `domain` and `assembly` variables are needed for `mono_jit_exec` call. If we would keep them separate, I would have to marshal them to java and then back to c (or save them somewhere and load them for the call). Or do you have a better suggestion how this could be handled?

Also `mono_droid_load_assembly` seemed like it is more related to execution of user code than runtime init.

Reply:
Based on the CoreCLR changes, I probably should move them to runtime init as suggested to be aligned.

Reply:
Moved to `mono_droid_runtime_init`